build/toolchain/linux/BUILD.gn

```diff
@@ -89,39 +149,90 @@ gcc_toolchain("clang_x64") {
   is_clang = true
 }
 
-gcc_toolchain("clang_arm64") {
-  if (use_clang_type_profiler) {
-    prefix = rebase_path("//third_party/llvm-allocated-type/Linux_x64/bin",
-                         root_build_dir)
-  } else {
-    prefix = rebase_path("//buildtools/linux-x64/clang/bin", root_build_dir)
+gcc_toolchain("x64") {
```

Comment:
This fills in some gaps, though we should _probably_ actually test this if we're going to add it as a supported toolchain.

/cc @zanderso for thoughts.